docs/arcadia-users-group/20230228-intro-to-python-1/lesson.ipynb

```diff
@@ -0,0 +1,884 @@
+{
```

Comment:
<p>I'd probably add a note about what version of Python we're using.</p><p> </p><p>Also is it possible to have this lesson in the Python shell? And do you plan to eventually show people ways to call python files from the shell? I don't want people to assume you can only do Jupyter notebooks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arcadia-Science/arcadia-computational-training/pull/50/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>I thought it would be a little easier for people to get comfortable with the immediacy of interactive computing - I wasn't planning on covering python calling from the shell but it's definitely important to learn. Maybe I can pin it to the end of this lesson? The other two lessons are pretty jam-packed with information. I'll write something up and also try to thread it through the next few lessons.</p>

Reply:
I added a section on running scripts from the command line, and changed the final problem in the problem set to have them write a script instead of a function!